src/main/java/com/google/sps/HomeBusiness.java

```diff
@@ -0,0 +1,62 @@
+package com.google.sps;
+
+import java.net.URL;    
+import java.util.Comparator;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Collection;
+import java.util.Collections;
+
+public final class HomeBusiness {
+
+  private String homeBusinessName;
```

Comment:
Agree with above! I also think it may be better to remove the 'home' prefix. Since we're only dealing with home businesses and not any other type of business, there isn't any ambiguity. HomeBusinessMockData is pretty long for a class name too.

Reply:
good suggestions! will incorporate.